src/scripts/transform-s3

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+#
+# S3 logs are plain text
+#
+
+set -e
+
+cat
```

Comment:
Instead of dummy transform scripts like this, `pull-s3` should probably just default to `cat` if the conventional `transform-{log-type}` script doesn't exist.